modules/ROOT/pages/message-flows.adoc

```diff
@@ -0,0 +1,36 @@
+= Message Flows
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To receive messages from a trading partner, create a message flow and configure the sections for the source, map, and target.
+Your organization is typically referred to as the host. The enterprise or customer you interact with is referred to as your partner.
+
+The source of the message flow is defined by the partner, a receiving endpoint at the host, often an API, and the message type you are receiving from the partner.
+
+The target is the final destination of a message flow and is defined by a host message type and endpoint, often a Host API.
+
+You'll need the following for the Source:
+
+* The partner's name and associated identifiers (such as X12-ISA, X12 GS, and so on).
+
+* The endpoint where you receive transmissions.
+
+* The message type to process for this message flow (such as X12, 4010, 850, and so on).
+
+* The endpoint where your partner may want to receive an acknowledgment related to the messages they send.
+
+* The DataWeave map that transforms the received message contents into a format you can use.
+
+You'll need the following for the Target:
+
+* The message type your system understands. This is also the format into which DataWeave map transforms received messages.
+
+* The endpoint in your organization where you route the transformed message.
+
+If a separate URL is required for returning MDNs, it is sent in the AS2 transmission and Anypoint Partner Manager reads it from there.
```

Comment:
Is this related to the previous bullet point, and is it part of what YOU are required to configure or is it outside of these requirements lists?